build.sbt

```diff
@@ -74,17 +74,9 @@ def altPublishSettings: Seq[Setting[_]] =
       val moduleSettings = (Keys.moduleSettings).value
       val ivy = new IvySbt((ivyConfiguration.value))
 
-      val module =
-        new ivy.Module(moduleSettings)
-      val newConfig =
-        new PublishConfiguration(
-          config.ivyFile,
-          altLocalRepoName,
-          config.artifacts,
-          config.checksums,
-          config.logging
-        )
-      streams.value.log.info("Publishing " + module + " to local repo: " + altLocalRepoName)
+      val module = new ivy.Module(moduleSettings)
+      val newConfig = config withResolverName altLocalRepoName withOverwrite false
```

Comment:
I am against the indiscriminate use of infix. Infixing should be limited to operator-like methods (e.g. collection library methods that accepts lambdas).

`config withResolverName altLocalRepoName withOverwrite false` is not readable. In my opinion, the only way this should be written is:

```scala
config
  .withResolverName(altLocalRepoName)
  .withOverwrite(false)
```